client/modules/IDE/components/About.jsx

```diff
@@ -43,7 +53,12 @@ function About(props) {
             target="_blank"
             rel="noopener noreferrer"
           >
-            <InlineSVG className="about__content-column-asterisk" src={asteriskUrl} alt="p5 asterisk" />
+            <InlineSVG
+              className="about__content-column-asterisk"
+              src={asteriskUrl}
+              alt="p5 asterisk"
```

Comment:
same, don't need an `alt` here, and the rest of the instance that have both title + alt